mxcube3/__init__.py

```diff
@@ -153,7 +153,11 @@ def complete_initialization(app):
         app.AUTO_MOUNT_SAMPLE = False
         app.AUTO_ADD_DIFFPLAN = False
         app.CENTRING_METHOD = CENTRING_METHOD.LOOP
-        app.NUM_SNAPSHOTS = 4
+        num_snapshots = app.collect.getProperty('num_snapshots')
+        if num_snapshots is not None:
```

Comment:
I think you can simply do

num_snapshots = app.collect.getProperty('num_snapshots', 4) 

to avoid the if ...

Reply:
thanks